charts/gateway-helm/values.tmpl.yaml

```diff
@@ -7,10 +7,10 @@ deployment:
     resources:
       limits:
         cpu: 500m
-        memory: 128Mi
+        memory: 1024Mi
       requests:
-        cpu: 10m
-        memory: 64Mi
+        cpu: 100m
```

Comment:
the `requests` does not need to increase. EG can run on cpu 10m and mem 64Mi. for the limits IMO can't fix memory hign usage. https://github.com/envoyproxy/gateway/issues/1613#issuecomment-1619288673

Reply:
I see, but request changing is unnecessary for my production env.

Reply:
I feel like `10m` is too low to start off with